pkg/utils/test_util.go

```diff
@@ -136,3 +140,66 @@ func (matcher AlreadyExistMatcher) FailureMessage(actual interface{}) (message s
 func (matcher AlreadyExistMatcher) NegatedFailureMessage(actual interface{}) (message string) {
 	return format.Message(actual, "not to be already exist")
 }
+
+// SchedulerWorkqueueKeyCollector helps collect keys from a scheduler work queue for testing
```

Comment:
could you please move this to test/utils?

Would like to deprecate this file and we should not mix testing utils and non testing utils in a package.

Reply:
Done.